erpnext/patches/v13_0/set_gratuity_payment_method.py

```diff
@@ -0,0 +1,8 @@
+from __future__ import unicode_literals
+import frappe
+
+def execute():
+    frappe.reload_doctype("Gratuity")
```

Comment:
```suggestion
    frappe.reload_doc("payroll", "doctype", "gratuity")
```

Reply:
@ruchamahabal I think both are the same..